demo/tests/test_conditions.py

```diff
@@ -0,0 +1,306 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from formidable import constants
+from formidable.models import PresetArg
+from formidable.forms import (
+    FormidableForm, fields, get_dynamic_form_class,
+    get_dynamic_form_class_from_schema
+)
+from formidable.forms.validations.presets import (
+    ConfirmationPresets
+)
+from formidable.serializers.forms import (
+    ContextFormSerializer, FormidableSerializer
+)
+
+
+class ConditionTestCase(TestCase):
+
+    def get_form_class(self, formidable, role):
+        return get_dynamic_form_class(formidable, role)
+
+    def setUp(self):
+        conditions_schema = [
+            {
+                'name': 'My Name',
+                'action': 'display_iff',
```

Comment:
probably, yes... it'll be less error-prone, especially when we'll have more than this action available.